doc/ntp/ntp-design.md

```diff
@@ -1057,6 +1091,33 @@ module sonic-ntp {
                                  server";
                 }
 
+                leaf minpoll {
+                    must "(current()/../maxpoll > current())" {
+                        error-message "maxpoll has to be larger than minpoll.";
```

Comment:
Can it be the same?

Reply:
Yes. They can be the same. Modified the doc.